packages/loopring_v3/circuit/Utils/Data.h

```diff
@@ -14,14 +16,130 @@ using json = nlohmann::json;
 namespace Loopring
 {
 
-enum class BlockType
-{
-    RingSettlement = 0,
+static auto dummySpotTrade = R"({
+    "fFillS_A": 0,
+    "fFillS_B": 0,
+    "orderA": {
+        "accountID": 0,
+        "allOrNone": false,
+        "amountB": "79228162514264337593543950335",
+        "amountS": "79228162514264337593543950335",
+        "buy": true,
+        "feeBips": 0,
+        "maxFeeBips": 0,
+        "orderID": "0",
+        "rebateBips": 0,
+        "tokenS": 0,
+        "tokenB": 1,
+        "validSince": 0,
+        "validUntil": 4294967295
+    },
+    "orderB": {
+        "accountID": 0,
+        "allOrNone": false,
+        "amountB": "79228162514264337593543950335",
+        "amountS": "79228162514264337593543950335",
+        "buy": true,
+        "feeBips": 0,
+        "maxFeeBips": 0,
+        "orderID": "0",
+        "rebateBips": 0,
+        "reduceOnly": 0,
+        "tokenS": 1,
+        "tokenB": 0,
+        "validSince": 0,
+        "validUntil": 4294967295
+    }
+})"_json;
+
+static auto dummyTransfer = R"({
+    "fromAccountID": 0,
+    "toAccountID": 2,
+    "amount": "0",
+    "fee": "0",
+    "feeTokenID": 0,
+    "tokenID": 0,
+    "validUntil": 4294967295,
+    "type": 0,
+    "ownerFrom": "0",
+    "to": "2",
+    "dualAuthorX": "0",
+    "dualAuthorY": "0",
+    "data": "0",
+    "payerToAccountID": 2,
+    "payerTo": "2",
+    "payeeToAccountID": 2,
+    "nonce": 0
+})"_json;
+
+static auto dummyWithdraw = R"({
+    "owner": "0",
+    "accountID": 0,
+    "tokenID": 0,
+    "amount": "0",
+    "feeTokenID": 0,
+    "fee": "0",
+    "to": "0",
+    "dataHash": "0",
+    "minGas": 0,
+    "type": 0
+})"_json;
+
+static auto dummyAccountUpdate = R"({
+    "owner": "0",
+    "accountID": 0,
+    "nonce": 0,
+    "publicKeyX": "13060336632196495412858530687189935300033555341384637843571668213752389743866",
+    "publicKeyY": "4915883150652842217472446614681036440072632592629277920562695676195366802174",
+    "walletHash": "0",
+    "feeTokenID": 0,
+    "fee": "0",
+    "type": 0
+})"_json;
+
+static auto dummyNewAccount = R"({
+    "payerAccountID": 0,
+    "feeTokenID": 0,
+    "fee": "0",
+    "nonce": 0,
+    "newAccountID": 2,
+    "newOwner": "1",
+    "newPublicKeyX": "13060336632196495412858530687189935300033555341384637843571668213752389743866",
+    "newPublicKeyY": "4915883150652842217472446614681036440072632592629277920562695676195366802174",
+    "newWalletHash": "0"
+})"_json;
+
+static auto dummyOwnerChange = R"({
+    "accountID": 0,
+    "feeTokenID": 0,
+    "fee": "0",
+    "newOwner": "0"
+})"_json;
+
+static auto dummyDeposit = R"({
+    "owner": "0",
+    "accountID": 0,
+    "tokenID": 0,
+    "amount": "0",
+    "index": "0"
+})"_json;
+
+static auto dummySignature = R"({
```

Comment:
what's the data signed, is this signature signed by the private key of the pubKey above?

Reply:
Not important, just some dummy data the circuits accepts, will never end up being used. At some places there are strict requirements that are checked in the circuit, and even if that logic isn't actually being used by a transaction (because we're only interested in another transaction type) it's still part of the circuit so it needs to be valid to make the full circuit valid.

Reply:
What valid really means for this signature? Which part of the code will fail if we change these values to 0s.

Reply:
It's a valid signature, just for different data. Before a signature is verified in the circuit it is first verified that the signature is indeed valid first (like checked with `PointValidator` in `EdDSA_Poseidon`). Providing 0s would fail that check, so we'd have to change `PointValidator` to not make it fail and just return if it is valid, but that's not always easy to do. So easier to just provide valid data in all cases.